dxr/lines.py

```diff
@@ -347,64 +343,94 @@ def finished_tags(text, refs, regions):
     remove_overlapping_refs(tags)
     return balanced_tags(tags)
 
+def tags_per_line(flat_tags):
+    """Split tags on LINE tags, yielding the tags of one line at a time
+        (no LINE tags are yielded)
 
-def es_lines(tags):
-    """Yield a list of dicts, one per source code line, that can be indexed
-    into the ``tags`` field of the ``line`` doctype in elasticsearch.
+    :arg flat_tags: An iterable of ordered, non-overlapping, non-empty tag
+        boundaries with Line endpoints at (and outermost at) the index of the
+        end of each line.
+    """
```

Comment:
To be nitpickingly consistent, we should really skip a line before the closing quotes on multi-line docstrings.